### PR TITLE
feat: make anarres PWA

### PR DESCRIPTION
**Pull request by teomrd**
Make Anarres installable with PWA features

Screen Shot 2023-02-24 at 18 54 40

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -28,11 +28,1665 @@
       },
       "devDependencies": {
         "autoprefixer": "^10.4.13",
+        "next-pwa": "^5.6.0",
         "postcss": "^8.4.20",
         "serve": "^14.2.0",
         "tailwindcss": "^3.2.4"
       }
     },
+    "node_modules/@ampproject/remapping": {
```
Comment (stathismor) [inline package-lock.json:37]: Damn, this thing adds a lot of dependencies. I saw it's about 50mb extra. So be it!

File: pages/_document.tsx [hunk 2]
```diff
@@ -9,6 +9,22 @@ export default function Document() {
           src="https://identity.netlify.com/v1/netlify-identity-widget.js"
         />
         <link rel="shortcut icon" href="/favicon.ico" />
+        <link rel="manifest" href="/manifest.json" />
+        <link rel="apple-touch-icon" href="/icon.png" />
+        <meta name="application-name" content="Anarres.fm" />
```
Comment (stathismor) [inline pages/_document.tsx:13]: Curious, did you use a tool to generate these? Images and code. Did you use the svg as a source? Should we put a comment maybe, or something at the README so we can remember how to do it next time?

**Review by stathismor — APPROVED** [review]
Amazeballs!

File: package-lock.json (hunk 1 above)
Comment (teomrd) [reply package-lock.json:37]: I know, but `next-pwa` seems to be the canonical way of building PWA nextJS applications. 
In any case the performance benefits are bigger, caching static resources etc

File: pages/_document.tsx (hunk 2 above)
Comment (teomrd) [reply pages/_document.tsx:13]: Good point. Added a readme section ;)

**Pull request merged**